packages/integrations/react/server.js

```diff
@@ -230,3 +230,11 @@ export default {
 	renderToStaticMarkup,
 	supportsAstroStaticSlot: true,
 };
+
+export const renderer = {
+	ssr: {
```

Comment:
This seems non-ideal, these things are exported already, why do it a second time with a different shape?

Reply:
I think this might be because this is the shape that the container accepts. But the `{ ssr: ... }` shape is completely internal Astro code. Renderers should not need be aware of it.

I would change the types of container's `renderers` prop to accept the non-nested shape as well, and then "fix" it internally. That way only container's internal code is aware of this issue.

Reply:
Instead of "fixing" it, I think the container should do just a simple check about the "shape" of the renderer passed. If it doesn't contain `check` and `renderToStaticMarkup`, then we can throw an error with some hints on how to fix the issue.